src/trigger.py

```diff
@@ -82,25 +45,78 @@ class cmd_run(Command):
     def __init__(self, sub_parser, name):
         super().__init__(sub_parser, name)
         self._logger = Logger("config/logger.conf", "trigger")
+        self._build_configs = None
+        self._db = None
+
+    def _log_revision(self, message, build_config, head_commit):
+        self._logger.log_message(
+            logging.INFO,
+            f"{message:32s} {build_config.name:32s} {head_commit}"
+        )
+
+    def _run_trigger(self, build_config, force):
+        head_commit = kernelci.build.get_branch_head(build_config)
+        node_list = self._db.get_nodes({
+            "revision.commit": head_commit,
+        })
+
+        if node_list:
+            if force:
```

Comment:
We can define `_force` in `cmd_run` class to avoid passing the variable in `_iterate_build_configs`, and `_run_trigger`.

`self._force = args.force`

Reply:
We could, but I don't think that's a good idea and that's why I'm passing it as an argument.  That way is about the run, whether it should force using a new revision or not.  The same object could be called several times.